inference-engine/src/preprocessing/ie_preprocess_gapi.cpp

```diff
@@ -983,15 +973,10 @@ bool PreprocEngine::preprocessBlob(const BlobTypePtr &inBlob, MemoryBlob::Ptr &o
 
     executeGraph(_lastComputation, batched_input_plane_mats, batched_output_plane_mats, batch_size,
         omp_serial, update);
-
-    return true;
 }
 
-bool PreprocEngine::preprocessWithGAPI(const Blob::Ptr &inBlob, Blob::Ptr &outBlob,
+void PreprocEngine::preprocessWithGAPI(const Blob::Ptr &inBlob, Blob::Ptr &outBlob,
         const ResizeAlgorithm& algorithm, ColorFormat in_fmt, bool omp_serial, int batch_size) {
-    if (!useGAPI()) {
-        return false;
-    }
 
```

Comment:
Please double check this method. For example in the line 999 you have the next code `return preprocessBlob(inNV12Blob, outMemoryBlob, algorithm, in_fmt, out_fmt, omp_serial,  batch_size);`

Reply:
`preprocessBlob` was changed to return `void` as well. So `return preprocessBlob(...)` is just short version of 
```c++
preprocessBlob();
return;
```